bucket/aircrack-ng.json

```
@@ -0,0 +1,38 @@
+{
+    "version": "1.6",
+    "description": "A suite of tools to assess WiFi network security",
+    "homepage": "https://www.aircrack-ng.org/",
+    "license": "GPL-2.0-only",
+    "architecture": {
+        "64bit": {
+            "url": "https://download.aircrack-ng.org/aircrack-ng-1.6-win.zip",
+            "hash": "sha1:ede4ac13ad04e9ec10b973460b36c92ce97829af"
+        }
+    },
+    "extract_dir": "aircrack-ng-1.6-win",
+    "pre_install": "if(!(Test-Path \"$persist_dir\\bin\\debug.log\")) {New-Item \"$dir\\bin\\debug.log\" | Out-Null}",
```

```suggestion
    "pre_install": "if(!(Test-Path \"$persist_dir\\bin\\debug.log\")) {New-Item \"$dir\\bin\\debug.log\" -ItemType File | Out-Null}",
```

If type is not specified, PS 7.1 creates a folder whereas PS 5.1 creates a file.